packages/calcite-components/src/components/block/block.tsx

```diff
@@ -86,7 +90,14 @@ export class Block
   /** When `true`, interaction is prevented and the component is displayed with lower opacity. */
   @property({ reflect: true }) disabled = false;
 
-  /** When `true`, displays a drag handle in the header. */
+  /** When `true`, the component is not draggable. */
+  @property({ reflect: true }) dragDisabled = false;
+
+  /**
+   * When `true`, the component displays a draggable button.
+   *
+   * @deprecated No longer necessary.
```

Comment:
Nitpick: Maybe we could beef up this deprecation message - something like "No longer necessary - use Block Group for draggable functionality" ?